### PR TITLE
New razor and susy clients

### PR DESCRIPTION
**Pull request by maojiajing**
Add client to HLT Razor paths.
To make efficiency plots automatically in dqm.
Relevant files:
DQMOffline/Trigger/python/RazorMonitor_Client_cff.py
DQMOffline/Trigger/python/SusyMonitoring_Client_cff.py
test output:
/afs/cern.ch/user/j/jmao/work/public/releases/RazorDQM/DQM_V0001_R000302523__Global__CMSSW_X_Y_Z__RECO.root

### CODE REVIEW

File: DQMOffline/Trigger/python/RazorMonitor_Client_cff.py [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+    #efficiencyProfile = cms.untracked.vstring(
+    #    "effic_Rsq_vs_LS 'Rsq efficiency vs LS; LS; Rsq efficiency' RsqVsLS_numerator RsqVsLS_denominator"
+    #),
+  
+)
+
+
+susyHLTRazorClient = cms.Sequence(
+    RazorEfficiency
+    #+ NoBPTXEfficiency
```
Comment (mtosi) [inline DQMOffline/Trigger/python/RazorMonitor_Client_cff.py]: can you delete this line ?

**Comment by mtosi** [comment]
@cmsbuild , please test

**Comment by mtosi** [comment]
ah !
I've just realized, this is for 92x
I think it is fine, @davidlange6 please comment if it is not the case

but we need anyhow the PR for the `master` and `93x` as well
@maojiajing please submit the mentioned PRs

**Comment by maojiajing** [comment]
@mtosi 
As for PR for 93X and master, which version of CMSSW use and which branch should I based on?
I tried CMSSW_9_3_0_pre1, it says it annot automatically merge even only the same two files changed. 
Thanks.

**Comment by mtosi** [comment]
on github you should chose
- CMSSW_9_3_X
- master

if your branch cannot be merged automatically,
then you need to prepare an other branch starting from
- the most recent 93x IB
- the most recent 94x IB

hope it helps
 mia

**Comment by maojiajing** [comment]
Hi Mia,

Based on the same development, I made another two PRs:
PR#20506 for 93x, 
PR#20508 for master(94x).

Your advice did help, thanks.

**Comment by mtosi** [comment]
;)
thanks !

**Comment by davidlange6** [comment]
Your PR is unmergeable. Please have a look and possibly rebase it.

**Comment by davidlange6** [comment]
CMSSW_9_2_X developement work is completed. Closing remaining PRs

**Pull request closed without merging**